### PR TITLE
Fix: gready attribute selector

### PR DESCRIPTION
**Pull request by bertyhell**

### CODE REVIEW

**Comment by JPeer264** [comment]
Hey @bertyhell thanks for your PR. Can you also add a test case in [replace.css.js](https://github.com/JPeer264/node-rcs-core/blob/master/test/replace.css.js) which covers your fix?

**Comment by bertyhell** [comment]
I couldn't get the tests to run without errors, so i just added the test code without running it.
If it doesn't work i'll just use my fork for now.

**Comment by bertyhell** [comment]
@JPeer264 Done
And I found a mistake in the fix because of the tests 🕵️

File: test/replace.css.js [hunk 1]
```diff
@@ -101,6 +101,13 @@ test('attribute selectors not renamed',
   { ignoreAttributeSelectors: true },
 );
 
+test('multiple attribute selectors without quotes',
```
[the remark this answers is not in the document]
Comment (bertyhell) [reply test/replace.css.js]: ok, has been added

**Comment by JPeer264** [comment]
Thanks @bertyhell 👍

**Comment by bertyhell** [comment]
@JPeer264 Could you create a release for 2.0.1, so I can start using this in my application?

**Comment by JPeer264** [comment]
Sure, it is published since it got merged.

**Pull request merged**